src/components/organisms/WorldGeneralForm/WorldGeneralForm.tsx

```diff
@@ -184,39 +212,36 @@ export const WorldGeneralForm: React.FC<WorldGeneralFormProps> = ({
             name="bannerImage"
             setValue={setValue}
             onChange={handleChange}
+            text="Upload highlight image"
           />
-        </AdminSection>
-        <AdminSection
-          title={
-            <>
-              Upload your logo &nbsp;
-              <span className="mod--subdued">(optional)</span>
-            </>
-          }
+        </InputGroup>
+
+        <InputGroup
+          title="Upload your logo"
           subtitle="A 400 px square image works best."
           withLabel
+          isOptional
         >
           <ImageInput
             name="logoImage"
             imgUrl={values?.logoImageUrl}
             error={errors.logoImageFile || errors.logoImageUrl}
             setValue={setValue}
             register={register}
-            small
+            text="Upload logo image"
             onChange={handleChange}
+            variant="round"
           />
-        </AdminSection>
+        </InputGroup>
+
         <FormErrors errors={errors} omitted={HANDLED_ERRORS} />
         <SubmitError error={error} />
 
         <AdminSidebarButtons>
-          <ButtonNG
-            className="AdminSidebarButtons__button--larger"
-            variant="primary"
-            {...saveButtonProps}
-          >
-            {worldId ? "Update" : "Create"}
-          </ButtonNG>
+          <Button {...saveButtonProps}>Save</Button>
```

Comment:
Spread here defeats the legacy `useMemo` above, might as well just remove the `useMemo` and put the few props here just like every other component.

Reply:
done: https://github.com/sparkletown/sparkle/pull/2868/commits/ba937c968da803ade57ed5627ca35f3baeade6a0